.github/workflows/aws-deploy.yml

```diff
@@ -9,6 +9,9 @@ on:
   push:
     branches: ['*']
 
+env:
+   CONTEXT: dev
```

Comment:
nice! lets make sure this is back ported to the https://github.com/Sage-Bionetworks/aws-application-deployment-template repo

suggestions:

* we are deploying from specific branches so don't we want to set this to the branch with [GITHUB_REF_NAME](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables)?
```
CONTEXT: ${{ env.GITHUB_REF_NAME }}
```

* how about updating below `cdk-deploy-dev:` to just `cdk-deploy:`?
* the `if` condition below (`if: ${{ github.ref == 'refs/heads/main' }}`) should be updated to reference the branch(s) that you plan to deploy from (i.e. dev/stage/prod/etc..)

Reply:
It looks like the variable is not `GITHUB_REF_NAME` but rather `REF_NAME`.  I did some experiments to determine this: https://github.com/brucehoff/conditional-env/actions/runs/4519535566/jobs/7960070640